### ISSUE TITLE
Consider supporting lang attribute

### ISSUE THREAD

**Issue by prowley** [lunarmodules/LDoc#339]
[W3C HTML](https://html.spec.whatwg.org/multipage/)

4.1.1 The html element
...
Authors are encouraged to specify a lang attribute on the root html element, giving the document's language. This aids speech synthesis tools to determine what pronunciations to use, translation tools to determine what rules to use, and so forth.

**Comment by alerque** [comment]
Where would you see this value originating from? A new config key?

**Comment by prowley** [comment]
I think that's the only real option. config/command line arg. I don't think anyone would likely set it from the default to be honest - it's not like your going to google translate the code comments :) It's really more about generating html that doesn't trip compliance checkers, which will likely spit something out about this.

**Comment by alerque** [comment]
I agree, and I'll try to look into this the next chance I get to hack on it. In the mean time PRs are always welcome!
